runtime/src/main/java/org/corfudb/runtime/collections/DynamicCorfuRecord.java

```diff
@@ -0,0 +1,54 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.DynamicMessage;
+import lombok.Getter;
+
+/**
+ * Contains the value as {@link CorfuDynamicMessage} after extracting it from {@link com.google.protobuf.Any}.
+ * This wrapper contains the payload and the respective typeUrl required to populate the Any on serialization.
+ * This object is stored as the value in the CorfuTable while viewing in the schema-less mode.
+ * <p>
+ * Created by zlokhandwala on 10/7/19.
+ */
+public class DynamicCorfuRecord {
+
+    /**
+     * TypeUrl generated for the value on serialization and populated in the Any message.
+     */
+    @Getter
+    private final String payloadTypeUrl;
+
+    /**
+     * Payload of the value stored in the CorfuTable.
+     */
+    public final CorfuDynamicMessage payload;
+
+    /**
+     * TypeUrl generated for the metadata on serialization and populated in the Any message.
+     */
+    @Getter
+    private final String metadataTypeUrl;
+
+    /**
+     * Payload of the metadata stored in the CorfuTable.
+     */
+    public final CorfuDynamicMessage metadata;
+
+    public DynamicCorfuRecord(String payloadTypeUrl,
+                              DynamicMessage payload,
+                              String metadataTypeUrl,
+                              DynamicMessage metadata) {
+        this.payloadTypeUrl = payloadTypeUrl;
+        this.payload = new CorfuDynamicMessage(payload);
+        this.metadataTypeUrl = metadataTypeUrl;
+        this.metadata = new CorfuDynamicMessage(metadata);
+    }
+
```

Comment:
Since this is the value part, I didn't think overriding equals and hashcode would be necessary.  Let me know if you still think it's required to maintain consistency.

Reply:
Done. Added.